### PR TITLE
Axioms for affine function bounds. Tests.

### PR DESCRIPTION
**Pull request by alex-ozdemir**
This PR is much larger, because it includes tests for the whole proof system.
You can run `lfscc sat.plf smt.plf lrat.plf drat.plf er.plf th_base.plf th_int.plf th_real.plf th_lira.plf th_lira_test.plf` from the `proofs/signatures` directory to run the tests (assuming `lfscc` is on your path).

This is the last signature PR!

### CODE REVIEW

File: proofs/signatures/th_lira_test.plf [hunk 1]
```diff
@@ -0,0 +1,339 @@
+; Depends On: th_lra.plf
```
Comment (yoni206) [inline proofs/signatures/th_lira_test.plf]:
```suggestion
; Depends On: th_lira.plf
```

File: proofs/signatures/th_lira_test.plf [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+           (bounded_aff_add _ _ _ _ _
+             (bounded_aff_mul_c _ _ _ 1/1 n3)
+             bounded_aff_ax_0_>=_0)))))
+  )))
+  )))
+  )))
+  ))
+)
+
+;; Term proof, 2 (>=), one (not >=)
```
Comment (yoni206) [inline proofs/signatures/th_lira_test.plf]: can you clarify this line? I think it is enough to do that once and the rest will be clear.

Comment (alex-ozdemir) [reply proofs/signatures/th_lira_test.plf]: Yeah, that is opaque. I've expanded this phrase.

**Comment by alex-ozdemir** [comment]
Thanks for reviewing, @yoni206! I've made changes to address the shortcomings you point out.

**Comment by yoni206** [comment]
Thanks for a great and fun-to-review sequence of PRs, that conforms with https://github.com/CVC4/CVC4/wiki/Developer-Guide#how-to-break-up-large-code-reviews !

**Pull request merged**